composer.json

```diff
@@ -29,7 +29,6 @@
     "pimcore/pimcore": "^11.0",
     "pimcore/search-query-parser": "^1.3",
     "pimcore/personalization-bundle": "^1.0",
```

Comment:
Do we need to update the pimcore version too?

Reply:
Yes, but as the whole topic is a potential BC break and note time critical for the moment I discussed with @fashxp that we will do it as new major release for 2025.1